.circleci/config.yml

```diff
@@ -205,7 +205,6 @@ workflows:
       - run-tests:
           name: run-tests-8.0
           php: "8.0.9"
-      # - run-tests:
-      #     name: run-tests-8.1
-      #     php: "8.1.0beta1"
-      #     composer: "--ignore-platform-reqs"
+      - run-tests:
```

Comment:
Adds 8.1 to test suite on CircleCI.

Reply:
Consider using [matrix jobs](https://circleci.com/blog/circleci-matrix-jobs/) if you are going to keep adding different versions for the same job.

Reply:
Didn't know about those, cool! I think I effectively reinvented that feature with my CircleCI config using variables, haha. I'll work on shifting over to those as time allows, much cleaner.